cli/script/command-parser.ts

```diff
@@ -273,7 +273,11 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
             .demand(/*count*/ 1, /*max*/ 2)  // Require one non-optional and one optional argument.
             .example("login", "Logs in to the CodePush server")
             .example("login --accessKey mykey", "Logs in on behalf of the user who owns and created the access key \"mykey\"")
+            .example("login --proxy proxyURL", "Logs in with the specified proxy url")
+            .example("login --noProxy", "Logs in without any proxy set in session file or environment vars")
```

Comment:
I would be in favor of omitting the noProxy example to keep things simple. I think showing the examples with and without a proxy server should accommodate the most common cases, and we can just rely on the docs and option description to explain the noProxy parameter.